tracking/src/DataCollection.js

```diff
@@ -0,0 +1,86 @@
+const NOT_AVAILABLE = "not available"
+class DataCollection {
+    constructor() {
+        console.log("DataCollection init"); 
+
+        this.getPlatform(); 
+        this.getLanguage(); 
+        this.getOS(); 
```

Comment:
are we able to get the OS version?

in the specs it is stated that we need to auto capture it as `osv` even though it is not a required field

Reply:
@swVisenze on hold on that, need to check with hung

@thehung111 from the backend, it seems like it will ignore some of those auto-capture params that is sent from the SDK (e.g platform,os,osv,device_brand,device_model)?
Because the backend will only look at the `user_agent` then there is no need for the SDK to try and retrieve those info?

Reply:
in the backend what happen is first server parse user agent and store device info. After that, the API parameters (osv, os,etc) in the controller level will override the ones parsed by user agent. This code is not in the filter.

Reply:
in the specs, for mobile device, we need to auto capture. For the web, it is optional.

Reply:
ic ic, then in that case, shiwei can actually remove quite abit of code here.

there should be no need to parse `platform,os,osv,device_brand,device_model,agent,agent_version` since it is already handled in the backend.

Just need to make sure the `User-Agent` in the header is not overridden by the user.

Reply:
removed data collection from user agent. since backend already handle this.